### ISSUE TITLE
How to contribute to this project

### ISSUE THREAD

**Issue by FrancisND** [smartstore/Smartstore#380]
Hi,

How do I contribute to this project.

Regards
Frank

**Comment by muratcakir** [comment]
Yes, it is open to the public and you are welcome to contribute.

**Comment by suprak** [comment]
> For best code quality we decided to port class by class and adapt the existing code to the new environment.

How do we jump in to help?
Is there a list of code that needs porting?

**Comment by moeddami** [comment]
It would be nice to have a gitter channel to communicate with project maintainers and also a contribution guidelines.
